MarkdownBear: Set NODE_PATH for remark-lint

jayvdb opened the pull request:
remark-cli needs to find its plugins.  When invoked in `/coala-bears`
directory, Node automatically looks in `node_modules`.  However if
the docker image is used normally the working directory will not
be `/coala-bears`.  Setting NODE_PATH ensures that the plugins can
be found.

Related to https://github.com/coala/coala-bears/issues/1235

jayvdb committed:
MarkdownBear: Set NODE_PATH for remark-lint

remark-cli needs to find its plugins.  When invoked in `/coala-bears`
directory, Node automatically looks in `node_modules`.  However if
the docker image is used normally the working directory will not
be `/coala-bears`.  Setting NODE_PATH ensures that the plugins can
be found.

Related to https://github.com/coala/coala-bears/issues/1235

underyx commented on Dockerfile:
```diff
@@ -4,7 +4,7 @@ MAINTAINER Fabian Neuschmidt fabian@neuschmidt.de
 
 # Set the locale
-ENV LANG=en_US.UTF-8 LANGUAGE=en_US:en PATH=$PATH:/root/pmd-bin-5.4.1/bin:/root/dart-sdk/bin:/coala-bears/node_modules/.bin:/root/bakalint-0.4.0:/root/elm-format-0.18
+ENV LANG=en_US.UTF-8 LANGUAGE=en_US:en PATH=$PATH:/root/pmd-bin-5.4.1/bin:/root/dart-sdk/bin:/coala-bears/node_modules/.bin:/root/bakalint-0.4.0:/root/elm-format-0.18 NODE_PATH=/coala-bears/node_modules
```
Could you break this down into one line per variable with `\`?

underyx commented:
So you found it! 🎉 

Awesome :D

underyx commented on .travis.yml:
```diff
@@ -30,6 +30,14 @@ script:
+    docker run -t -i coala-docker /bin/sh -c "
+      cd /tmp;
+      echo foo > foo.md;
+      coala-ci -I --bears MarkdownBear --files foo.md;
```
Isn't `coala-ci` deprecated? I think it should be `coala --non-interactive` instead.

underyx replied:
Also I'm not sure what `-I` is, please use the long-form version? 😇

jayvdb committed:
.travis.yml: Test another Mercurial repository

The repository on selenic.com is timing out.
Use an empty repository on BitBucket which should be more
reliable.

Fixes https://github.com/coala/docker-coala-base/issues/160

sils commented:
ack https://github.com/coala/docker-coala-base/pull/159/commits/499223554ff686e540c53badbfa45ccee07807a4

sils commented:
ack https://github.com/coala/docker-coala-base/pull/159/commits/3540ab3fa0b259b1bd5ff3b8b489ec6da7d2b2da

sils commented:
@rultor merge

rultor commented:
> @rultor merge

@sils OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/11430-295155082)

rultor commented:
> @rultor merge

@sils Done! FYI, the full log is [here](http://www.rultor.com/t/11430-295155082) (took me 1min)
